tests/classifiersFrameworks/test_tensorflow.py

```diff
@@ -23,10 +23,15 @@
 from art.estimators.classification.tensorflow import TensorFlowV2Classifier
 from art.defences.preprocessor.spatial_smoothing import SpatialSmoothing
 from art.defences.preprocessor.spatial_smoothing_tensorflow import SpatialSmoothingTensorFlowV2
-from art.attacks.evasion import FastGradientMethod
+from art.attacks.evasion import FastGradientMethod, ProjectedGradientDescent
 
 from tests.attacks.utils import backend_test_defended_images
 from tests.utils import ARTTestException
+import art.estimators.classification
+import art
+import sklearn
+import sklearn.datasets
+import sklearn.model_selection
```

Comment:
To follow PEP8, could you please move the imports from `sklearn` to line 22 and the import from `art` to line 27?

Reply:
Completed, hopefully is as expected. If not let us know.